src/binary/non_blocking/raw_binary_reader.rs

```diff
@@ -1030,9 +996,7 @@ impl<'a> Iterator for AnnotationsIterator<'a> {
 /// In this way, the RawBinaryBufferReader will never be in a bad state. It only updates when the
 /// TxReader has already found the next item.
 struct TxReader<'a, A: AsRef<[u8]>> {
-    ion_version: (u8, u8),
     state: &'a mut ReaderState,
-    main_buffer: &'a BinaryBuffer<A>,
```

Comment:
🗺️ Now that the tx_buffer below has the original buffer's `total_consumed`, this field has been removed.